ncm-spma/src/main/pan/components/spma/functions-rpm.pan

```diff
@@ -0,0 +1,572 @@
+# ${license-info}                                                                                                                                                                                                                            
```

Comment:
The functions seem to be the same for the current 'yum' spma plugin as well as for other plugins as 'yumng'. So just value of packager should be sufficient IMO.